src/components/PatentUpload/PatentUpload.js

```diff
@@ -128,8 +133,9 @@ class PatentUpload extends Component {
                     <label className="toUploadLabel" for="toUploadLabel">Patents to be Upload: </label>
                     <div className="fileList" id="fileList" ref={ref => (this.fileList = ref)}></div>
                 </div>
-                <button className="uploadButton" onClick={this.uploadPatent}> Upload Files </button>
-
+                
+                {!this.state.loading ? <button className="uploadButton" onClick={this.uploadPatent}> Upload Files </button> : null }
```

Comment:
nit: can we do it in same conditional? Render spinner instead of `null` in the else case?